### DISCUSSION TITLE
Why .envs/ is copied into docker image during build?

### DISCUSSION THREAD

**Discussion by zhaoruibing** [cookiecutter/cookiecutter-django#4474] [Q&A] [answered]
Hi,

Since `.envs/` is not included in [.dockerignore](https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/.dockerignore), and there is `COPY . ${APP_HOME}` in Dockerfile.
`compose build django` will copy environment files into the image.

Environment files contain secrets, which I thought should not be included in the image.
Besides, the environment files are already passed to the container use compose file's `env_file`.

Is there a particular reason for doing this? Thanks.

**Comment by browniebroke** [comment]
That sounds bad... I don't think it's intentional, so it's probably a bug.

**Reply by zhaoruibing** [reply]
Thanks for maintaining cookiecutter-django, it has been a pleasant to use.

Given my limited knowledge to this project, I was assuming this was needed for CI or testing.
If this is not indented, would this better be addressed as a Github issue? In case someone/something relies on this behavior.

I have added `.envs/` to my own .dockerignore, and everything works.
My project is not production yet, still in local development stage.

**Reply by browniebroke** [reply, marked as answer]
Yes, a GitHub issue or directly via a pull request that adds the missing entry to the `.dockerignore`, and we can discuss/test further on it directly. If you don't know where to start, you can look at our [contributing guide](https://github.com/cookiecutter/cookiecutter-django/blob/master/CONTRIBUTING.md).

**Reply by zhaoruibing** [reply]
[Pull request 4476](https://github.com/cookiecutter/cookiecutter-django/pull/4476) has been created.
I will close this discuss.
